tools/astarte_import/lib/astarte/import/populatedb.ex

```diff
@@ -48,8 +48,8 @@ defmodule Astarte.Import.PopulateDB do
     nodes = Application.get_env(:cqerl, :cassandra_nodes)
     {host, port} = Enum.random(nodes)
     Logger.info("Connecting to #{host}:#{port} cassandra database.", realm: realm)
-
-    {:ok, conn} = Database.connect(realm)
+    opts = [cassandra_nodes: nodes, realm: realm]
+    {:ok, conn} = Database.connect(opts)
     {:ok, xandra_conn} = Xandra.start_link(nodes: ["#{host}:#{port}"])
```

Comment:
This might be ok, but I think that on another PR we should remove `cqerl` and just use Xandra in the whole application

Reply:
Ok, I will open new PR for remove call in application. Can we merge this code.

Reply:
Yes! But we should entirely remove cqerl, not just take it away from this file